playkit/src/main/java/com/kaltura/playkit/backend/base/FormatsHelper.java

```diff
@@ -0,0 +1,102 @@
+package com.kaltura.playkit.backend.base;
+
+import android.support.annotation.StringDef;
+
+import com.kaltura.playkit.PKDrmParams;
+import com.kaltura.playkit.PKMediaFormat;
+import com.kaltura.playkit.backend.base.data.BasePlaybackSource;
+
+import java.lang.annotation.Retention;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.annotation.RetentionPolicy.SOURCE;
+
+/**
+ * Created by tehilarozin on 13/02/2017.
+ */
+
+public class FormatsHelper {
+
+    @Retention(SOURCE)
+    @StringDef(value = {FormatName.MpegDash, FormatName.MpegDashDrm, FormatName.AppleHttp,
+            FormatName.Url, FormatName.UrlDrm})
+    public @interface FormatName {
```

Comment:
to enable the use of string constants instead of enums, and keep type safe values 
see: https://noobcoderblog.wordpress.com/2015/04/12/java-enum-and-android-intdefstringdef-annotation/
i put the constant values inside the @interface to enclose it, they can be pulled out or it can be changed
to enum... your decision